modules/apps/frontend-token/frontend-token-definition-test/src/testIntegration/java/com/liferay/frontend/token/definition/test/FrontendTokenDefinitionRegistryTest.java

```diff
@@ -90,37 +90,48 @@ public void tearDown() throws PortalException {
 
 	@Test
 	public void testGetFrontendTokenDefinitions() {
+		List<FrontendTokenDefinition> frontendTokenDefinitionsList =
+			_frontendTokenDefinitionRegistry.getFrontendTokenDefinitions(
+				_layoutSet.getCompanyId());
+
+		Assert.assertTrue(
+			ListUtil.exists(
+				frontendTokenDefinitionsList,
+				item -> Objects.equals(
+					item.getThemeId(), _THEME_ID_CLIENT_EXTENSION_ENTRY)));
+
 		Assert.assertTrue(
 			ListUtil.exists(
-				_frontendTokenDefinitionRegistry.getFrontendTokenDefinitions(
-					_layoutSet.getCompanyId()),
-				frontendTokenDefinition -> Objects.equals(frontendTokenDefinition.getThemeId(), _THEME_ID_CLIENT_EXTENSION_ENTRY)));
+				frontendTokenDefinitionsList,
+				item -> Objects.equals(
```

Comment:
I'd rename `item` to `frontendTokenDefinition`

Reply:
oh done